### PR TITLE
Move SIWA for Web button to the Sign In Page

### PR DESCRIPTION
**Pull request by coldlink**
## Changes
- Added Apple as an additional OAuth provider
- Sourced Apple logo from [Apple Design Resources](https://developer.apple.com/design/resources/)
- Update text, styles to match other OAuth buttons
- Add link to Federation API on button

Screenshot 2020-03-25 at 16 12 20

## Checklist

- [x] Move button on to sign in page
- [x] Test flow works on CODE
- [x] Test flow works on PROD

### CODE REVIEW

File: app/com/gu/identity/frontend/models/text/OAuthText.scala [hunk 1]
```diff
@@ -41,13 +49,15 @@ object OAuthSignInText {
   def apply()(implicit messages: Messages): OAuthSignInText =
     OAuthSignInText(
       facebook = messages("oauth.signIn", facebook),
-      google = messages("oauth.signIn", google)
+      google = messages("oauth.signIn", google),
+      apple = messages("oauth.signIn", google)
```
Comment (vlbee) [inline app/com/gu/identity/frontend/models/text/OAuthText.scala]: Line 53: should this be ", apple)"?

**Comment by vlbee** [comment]
Just adding that this PR is not to be merged until we are ready to go live with SIWA.

File: app/com/gu/identity/frontend/models/text/OAuthText.scala (hunk 1 above)
Comment (coldlink) [reply app/com/gu/identity/frontend/models/text/OAuthText.scala]: woops >.<

**Comment by lmath** [comment]
I merged in master just because this was raised a while ago. I then deployed the branch to code and then:
- signed in with Apple on CODE
- removed my SC_GU_LA cookie
- tried to navigate to my newsletters page
- saw that this directed to /reauthenticate and that the SIWA button appeared
- signed in with Apple again

it's cool that now /signin and /reauthenticate look the same now 😎 
image

**Pull request merged**